src/Controls/tests/Core.UnitTests/BindingUnitTests.cs

```diff
@@ -2041,51 +2041,59 @@ public virtual void OnPropertyChanged([CallerMemberName] string propertyName = n
 			}
 		}
 
-		[Fact]
+		[Fact, Category(TestCategory.Memory)] 
 		public async Task BindingUnsubscribesForDeadTarget()
```

Comment:
This one seems to be failing now 

https://dev.azure.com/xamarin/public/_build/results?buildId=97433&view=ms.vss-test-web.build-test-results-tab&runId=2130495&resultId=105355&paneView=debug